### DISCUSSION TITLE
[Closed] Ajout d'un import d'animations DragonBones

### DISCUSSION THREAD

**Discussion by mirobane** [skarab42/marv#90] [Ideas]
### Hello ! 

l'idée, comme son nom l'indique, l'idée serait d'ajouter une option d'import de fichiers d'animation DragonBones : 
doc : https://github.com/DragonBones/DragonBonesJS

pour lire des animations temps réel et les lier a quelques trigers d'interaction dans marv' 

anim 1 : idle
anim 2 : new follow
ect ect

exemples de résultats : https://www.youtube.com/watch?v=8KSgftGDx20&list=PLAxFOjtO8N8OG-1N6MMH2SNc8659UdhmM&index=3

Sample de fichier d'animation 
[DragonBonesSampleBear.zip](https://github.com/skarab42/marv/files/5859898/DragonBonesSampleBear.zip)

**résumé de la réponse live : ne sera pas ajouté a marv'. plus le penser via le futur système de plugins car feature trop spécifique**

**Comment by skarab42** [comment]
Pas dans une version officiel, mais surement possible via le "futur" système de modules ;)
